code/datums/martial/psychotic_brawl.dm

```diff
@@ -3,69 +3,175 @@
 	id = MARTIALART_PSYCHOBRAWL
 
 /datum/martial_art/psychotic_brawling/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
-	if(HAS_TRAIT(A, TRAIT_PACIFISM))
-		return FALSE
-	return psycho_attack(A,D)
+	return psycho_disarm(A,D)
 
 /datum/martial_art/psychotic_brawling/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
-	if(HAS_TRAIT(A, TRAIT_PACIFISM))
-		return FALSE
-	return psycho_attack(A,D)
+	return psycho_grab(A,D)
 
 /datum/martial_art/psychotic_brawling/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
```

Comment:
Shouldn't the `harm_act()` check for pacifism since it can actually do damage?

Reply:
Harm_act checks for pacifism by default